README.md

```diff
@@ -39,14 +37,45 @@ To run:
 
 
 ### analysis.py
-Script that can be used to:
-1. print total snf amount and isotope mass (analysis.snf(cursor))
-2. create stacked bar chart of power capacity and number of reactors per region (analysis.plot_power(cursor))
-3. create a multi-line plot chart of waste isotope per time (analysis.isotope_vs_time(cursor))
 
-The following command on the terminal will execute all three:
+Input : CYCLUS output file (.sqlite)
+`python analysis.py [outputfile]`
+
+Most functions return a dictionary of lists (timeseries of a value)
+that can be used to plot a stacked bar chart or a line plot.
+
+Major analysis functions include:
```

Comment:
I have made basic unit tests for some functions,

but most functions have to do with sqlite files (anything with input cursor).
Would having a sample sqlite file and using that to run unit tests be the right way of doing it?